cmd/crates/soroban-test/tests/it/integration/emulator.rs

```diff
@@ -0,0 +1,83 @@
+use stellar_ledger::{Blob, Error};
+
+use soroban_test::{AssertExt, TestEnv, LOCAL_NETWORK_PASSPHRASE};
+use std::sync::Arc;
+
+use soroban_cli::xdr::{
+    self, Memo, MuxedAccount, Operation, OperationBody, PaymentOp, Preconditions, SequenceNumber,
+    Transaction, TransactionExt, Uint256,
+};
+
+use stellar_ledger::emulator_test_support::*;
+
+use test_case::test_case;
+
+use crate::integration::util::{deploy_contract, DeployKind, HELLO_WORLD};
+
+// #[test_case("nanos"; "when the device is NanoS")]
```

Comment:
I was running into issues around the server locally. Since all the accounts share the same seed phrase it was acting as one account. I also tried updating the testcase to add an hd-path, but it was causes issues. So I just wanted one to work on CI and then was going to return to it.